library/lua/gui/dialogs.lua

```diff
@@ -150,6 +150,8 @@ ListBox.focus_path = 'ListBox'
 
 ListBox.ATTRS{
     with_filter = false,
+    dismiss_on_submit = true,
+    dismiss_on_submit2 = true,
```

Comment:
from the perspective of the ListBox, shouldn't these be named `dismiss_on_select` and `dismiss_on_select2`?

Reply:
I probably got confused because of how the Lua API docs mention on_submit/2, but they are actually not directly accessible anymore. I'll change it to (maybe) avoid further confusion.